assets/src/edit-story/app/layout/index.js

```diff
@@ -34,11 +34,6 @@ const Editor = styled.div`
 	line-height: ${ ( { theme } ) => theme.fonts.body1.lineHeight };
 	letter-spacing: ${ ( { theme } ) => theme.fonts.body1.letterSpacing };
 	background-color: ${ ( { theme } ) => theme.colors.bg.v1 };
-	position: absolute;
```

Comment:
If we remove `position: absolute`, then we should probably set it to `position:relative`.